src/main/resources/lib/ownership/cellWrapper.jelly

```diff
@@ -0,0 +1,20 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define">
+  <st:documentation>
+    The wrapper will be a `td` tag on Jenkins Core less than 2.264, and a `div` tag after that
+  </st:documentation>
```

Comment:
```suggestion
    The wrapper will be a <code>div</code> tag on Jenkins pages with the <code>divBasedFormLayout</code> flag, and a <code>td</code> tag otherwise
  </st:documentation>
```